erts/emulator/Makefile.in

```diff
@@ -625,7 +631,24 @@ $(TTF_DIR)/driver_tab.c: Makefile.in utils/make_driver_tab
 	$(gen_verbose)LANG=C $(PERL) utils/make_driver_tab -o $@ -nifs $(NIF_OBJS) $(STATIC_NIF_LIBS) -drivers $(DRV_OBJS) $(STATIC_DRIVER_LIBS)
 GENERATE += $(TTF_DIR)/driver_tab.c
 
-
+beam/erl_db_insert_list.inc: $(YCF_EXECUTABLE) beam/erl_db.c
+	$(YCF_EXECUTABLE) \
```

Comment:
Can we prefix this with `$(gen_verbose)`?

Reply:
Yes! See commit below.